tidb-cloud/serverless-driver-node-example.md

```diff
@@ -0,0 +1,95 @@
+---
+title: TiDB Cloud Serverless Driver Node.js Tutorial
+summary: Learn how to use TiDB Cloud serverless driver in a local Node.js project.
+---
+
+# TiDB Cloud Serverless Driver Node.js Tutorial
+
+Learn how to use TiDB Cloud serverless driver in a local Node.js project.
+
+> **Note:**
+>
+> - This tutorial is applicable to TiDB Serverless clusters only. 
+> - To learn how to use TiDB Cloud serverless driver with Cloudflare Workers, Vercel Edge Functions, and Netlify Edge Functions, check out our [Insights into Automotive Sales](https://car-sales-insight.vercel.app/) and the [sample repository](https://github.com/tidbcloud/car-sales-insight).
+
+## Before you begin
+
+To complete this step-by-step tutorial, you need the following:
+
+- The [Node.js](https://nodejs.org/en) >= 18.0.0.
+- The [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) or your preferred package manager.
+- A TiDB Serverless cluster. If you don't have any, you can [create a TiDB Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md).
+
+## Step 1. Create a local Node.js project
+
+1. Create a project named `node-example`:
+
+    ```
+    mkdir node-example
+    cd node-example
+    ```
+
+2. Install the TiDB Cloud serverless driver using npm or your preferred package manager.
+
+    The following command takes installation with npm as an example. Executing this command will create a `node_modules` directory and a `package.json` file in your project directory.
+
+    ```
+    npm install @tidbcloud/serverless
+    ```
+
+## Step 2. Use the serverless driver
+
+The serverless driver supports both CommonJS and ESM modules. The following steps take the usage of the ESM module as an example.
```

Comment:
A bit weird to say "ESM Module" since ESM = **E**CMA**S**cript **M**odule. It's either **ESM** or **ES Module**.
```suggestion
The serverless driver supports both CommonJS and ES modules. The following steps take the usage of the ES module as an example.
```

Reply:
ESM means ES modules I think.